nomad/integration-tests-new.nomad

```diff
@@ -125,7 +138,11 @@ job "integration-tests-new" {
         AWS_REGION = var.aws_region
 
         RUST_BACKTRACE = 1
-        RUST_LOG       = var.rust_log
+        RUST_LOG       = local.log_level
+
+        GRAPL_LOG_LEVEL = local.log_level
+
+        MG_ALPHAS = "localhost:${NOMAD_UPSTREAM_PORT_dgraph-alpha-0-grpc-public}"
```

Comment:
i believe a full NOMAD_UPSTREAM_ADDR would work here in lieu of localhost:

but NBD either way

Reply:
fixed